pkgs/by-name/co/color-scheme-generator/package.nix

```
@@ -0,0 +1,44 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+  perl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "color-scheme-generator";
+  version = "0.9.2.1";
+
+  src = fetchFromGitHub {
+    owner = "nikolaizombie1";
+    repo = "color_scheme_generator";
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```
This is a new argument for `fetchFromGitHub` that we should use instead when referring to tags (as to avoid branches of the same name, for example)